systemtest/src/test/java/io/strimzi/systemtest/security/SecurityST.java

```diff
@@ -1193,21 +1159,13 @@ void testCaRenewalBreakInMiddle() {
                 .build()
         );
 
-        // Configuration `acks=all` paired with Kafka's `min.insync.replicas=2` enforces there are at least 2 replicas with all produced data before test continues.
-        String producerAdditionConfiguration = "acks=all\n";
-        KafkaClients kafkaClients = new KafkaClientsBuilder()
-            .withTopicName(testStorage.getTopicName())
-            .withMessageCount(testStorage.getMessageCount())
-            .withBootstrapAddress(KafkaResources.tlsBootstrapAddress(testStorage.getClusterName()))
-            .withProducerName(testStorage.getProducerName())
-            .withConsumerName(testStorage.getConsumerName())
-            .withNamespaceName(testStorage.getNamespaceName())
+        // (implicit) Configuration `acks=all` paired with Kafka's `min.insync.replicas=2` enforces there are at least 2 replicas with all produced data before test continues.
```

Comment:
Hmm do we need this comment when you removed the additional config? I guess this will be more confusing than removing it.

Reply:
i put it there mainly to underline fact that the config is implicit otherwise rest of the config would not make sense, but yeah as this is rather trivial part of Kafka i agree that removing is better move.